### ISSUE TITLE
Vagrant 2.2.6 doesn't support VirtualBox 6.1.0

### ISSUE THREAD

**Issue by nickgryg** [hashicorp/vagrant#11276]
### Summary

Vagrant 2.2.6 stopped working after Virtualbox was updated to version 6.1.0.

### Vagrant version
Vagrant 2.2.6

### Host operating system
macOS Catalina Version 10.15.2 (19C57)

### Guest operating system
CentOS 7

### Vagrantfile
```ruby
Vagrant.configure("2") do |config|
  config.vm.box = "centos/7"
  config.vm.network "private_network", ip: "192.168.33.10"
  config.vm.provider "virtualbox" do |vb|
      vb.memory = "1024"
  end
end
```

### Debug output
```
% vagrant up
The provider 'virtualbox' that was requested to back the machine
'default' is reporting that it isn't usable on this system. The
reason is shown below:

Vagrant has detected that you have a version of VirtualBox installed
that is not supported by this version of Vagrant. Please install one of
the supported versions listed below to use Vagrant:

4.0, 4.1, 4.2, 4.3, 5.0, 5.1, 5.2, 6.0

A Vagrant update may also be available that adds support for the version
you specified. Please check www.vagrantup.com/downloads.html to download
the latest version.
```

### Expected behavior
`vagrant up` works

### Actual behavior
`vagrant up` doesn't work

### Steps to reproduce
1. Install the latest Virtualbox 6.1.0
2. Install the latest Vagrant 2.2.6
3. Try to use Vagrant

### Additional information about the packages
```
% brew cask info virtualbox
virtualbox: 6.1.0,135406
https://www.virtualbox.org/
/usr/local/Caskroom/virtualbox/6.1.0,135406 (3 files, 123.2MB)
From: https://github.com/Homebrew/homebrew-cask/blob/master/Casks/virtualbox.rb
==> Name
Oracle VirtualBox
==> Artifacts
VirtualBox.pkg (Pkg)
==> Caveats
To install and/or use virtualbox you may need to enable its kernel extension in:
  System Preferences → Security & Privacy → General
For more information refer to vendor documentation or this Apple Technical Note:
  https://developer.apple.com/library/content/technotes/tn2459/_index.html
```

```
% brew cask info vagrant
vagrant: 2.2.6
https://www.vagrantup.com/
/usr/local/Caskroom/vagrant/2.2.6 (3 files, 31.7MB)
From: https://github.com/Homebrew/homebrew-cask/blob/master/Casks/vagrant.rb
==> Name
Vagrant
==> Artifacts
vagrant.pkg (Pkg)
```

**Comment by jbonhag** [comment]
Hi there,

Support for VirtualBox 6.1 has not shipped yet and will be included in the next release. Downgrading VirtualBox to 6.0.x until the next release is available will resolve this issue.

Cheers!

Duplicate of #11249

**Comment by RogerE** [comment]
Unfortunately VirtualBox 6.0 is not available for Ubuntu 19.10, so it would be great if the next release could be issued soon.

**Comment by chrisroberts** [comment]
@RogerE The next release is coming soon (within the next few days). We have been in the process of updating our release pipeline which is just about complete. The next release will be pushed out as soon as everything has been finalized. 

Cheers!

**Comment by RogerE** [comment]
Thanks @chrisroberts, appreciated!

**Comment by eurico-goncalves** [comment]
@RogerE You can also use this workaround while we all wait for a new release:
https://blogs.oracle.com/scoter/getting-vagrant-226-working-with-virtualbox-61-ga

I've tested it myself and works great!

**Comment by RogerE** [comment]
@eurico-goncalves thanks for the tip, but the release of Virtualbox 6.0.16 a few days ago solved it for me since that works with Ubuntu 19.10.
